test/unit/test_configmanager.py

```diff
@@ -611,3 +613,60 @@ def test_configoption_missing_nest_path():
             _nest_path=None,
             _root_manager=ConfigManager(name="test_manager"),
         )
+
+
+def test_configoption_default_value(tmp_path, monkeypatch):
+    env_name = random_string(
+        5,
+        "SF_TEST_OPTION_",
+        choices=string.ascii_uppercase,
+    )
+    conf_val = random_string(5)
+    cm = ConfigManager(
+        name="test_manager",
+        file_path=tmp_path / "config.toml",
+    )
+    cm.add_option(
+        name="test_option",
+        env_name=env_name,
+        default=conf_val,
+    )
+    assert cm["test_option"] == conf_val
+    env_value = random_string(5)
+    with monkeypatch.context() as c:
+        c.setenv(env_name, env_value)
+        assert cm["test_option"] == env_value
+
+
+def test_defaultconnectionname(tmp_path, monkeypatch):
+    with monkeypatch.context() as m:
+        m.delenv("SNOWFLAKE_DEFAULT_CONNECTION_NAME", raising=False)
+        assert CONFIG_PARSER["default_connection_name"] == "default"
+    env_val = random_string(5, "DEF_CONN_")
+    with monkeypatch.context() as m:
+        m.setenv("SNOWFLAKE_DEFAULT_CONNECTION_NAME", env_val)
+        assert CONFIG_PARSER["default_connection_name"] == env_val
+    assert CONFIG_PARSER.file_path is not None
+    con_name = random_string(5, "conn_")
+    c_file = tmp_path / "config.toml"
+    c_file.write_text(
+        dedent(
+            f"""\
+            default_connection_name = "{con_name}"
+
+            [connections.{con_name}]
```

Comment:
So the file content can also be like below?
```
[connections.default]
user="testuser"
account="testaccount"
password="testpassword"
```

Reply:
That's right!